### PR TITLE
Feat: slinky's market map generation in genesis

### PR DESCRIPTION
**Pull request by pr0n00gler**
Slinky's market map generation in genesis

Related PR: 
* https://github.com/neutron-org/neutron-integration-tests/pull/324

### CODE REVIEW

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/9818715409

**Review by NeverHappened — DISMISSED** [review]
nice!

File: network/init-neutrond.sh [hunk 1]
```diff
@@ -719,12 +725,27 @@ function convert_bech32_base64_esc() {
...
+MARKETS=$(cat markets.json)
+
+NUM_MARKETS=$(echo "$MARKETS" | jq '.markets | length + 1')
+
+NUM_MARKETS=$NUM_MARKETS; jq --arg num "$NUM_MARKETS" '.app_state["oracle"]["next_id"] = $num' "$GENESIS_PATH" > genesis_tmp.json && mv genesis_tmp.json "$GENESIS_PATH"
+MARKETS=$MARKETS; jq --arg markets "$MARKETS" '.app_state["marketmap"]["market_map"] = ($markets | fromjson)' "$GENESIS_PATH" > genesis_tmp.json && mv genesis_tmp.json "$GENESIS_PATH"
+MARKETS=$MARKETS; jq --arg markets "$MARKETS" '.app_state["oracle"]["currency_pair_genesis"] += [$markets | fromjson | .markets | values | .[].ticker.currency_pair | {"currency_pair": {"Base": .Base, "Quote": .Quote}, "currency_pair_price": null, "nonce": 0} ]' "$GENESIS_PATH" > genesis_tmp.json && mv genesis_tmp.json "$GENESIS_PATH"
+MARKETS=$MARKETS; jq --arg markets "$MARKETS" '.app_state["oracle"]["currency_pair_genesis"] |= (to_entries | map(.value += {id: (.key + 1)} | .value))' "$GENESIS_PATH" > genesis_tmp.json && mv genesis_tmp.json "$GENESIS_PATH"
+
+cat markets.json
```
Comment (swelf19) [inline network/init-neutrond.sh]: debug line?

File: network/init-neutrond.sh [hunk 2]
```diff
@@ -53,6 +53,12 @@ CW4_GROUP_CONTRACT=$THIRD_PARTY_CONTRACTS_DIR/cw4_group.wasm
 
 NEUTRON_CHAIN_MANAGER_CONTRACT=$CONTRACTS_BINARIES_DIR/neutron_chain_manager.wasm
 
+# Slinky genesis configs
+USE_CORE_MARKETS=true
```
Comment (swelf19) [inline network/init-neutrond.sh]: may be we can set those as env variables with a default values? For easy configuration without editing the script
`USE_CORE_MARKETS=${USE_CORE_MARKETS:-true}

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/9818715409

**Pull request merged**